AEPMessaging/Sources/Messaging.swift

```diff
@@ -223,12 +223,6 @@ public class Messaging: NSObject, Extension {
             return
         }
 
-        // hard dependency on configuration shared state
-        guard let configSharedState = getSharedState(extensionName: MessagingConstants.SharedState.Configuration.NAME, event: event)?.value else {
-            Log.debug(label: MessagingConstants.LOG_TAG, "Event processing is paused, waiting for valid configuration - '\(event.id.uuidString)'.")
-            return
-        }
```

Comment:
why did we remove this?

Reply:
This was not necessary `configSharedState ` variable was never used and each type of event retrives config/edge xdm shared state inside their method while being processed.